### PR TITLE
Logging

### PR DESCRIPTION
**Pull request by AledWatkins**
Added some logging models, some limited logging functionality to the binance client and a view for listing the logs

I thought it would be good to get some logging in early since we are propbably going to need to refer to them a lot later down the line.
Not sure if this is a bit overkill but I think it will be nice to have logs that are saved to the DB.

### CODE REVIEW

File: af_trading_bot/core/binance_client.py [hunk 1]
```diff
@@ -2,18 +2,52 @@
 
 from django.conf import settings
 from binance import client as binance_client
+from core import logging, models
 
 
 @contextlib.contextmanager
 def initiate_client():
     yield binance_client.Client(settings.BINANCE_CLIENT_KEY, settings.BINANCE_CLIENT_SECRET_KEY)
```
Comment (DanielOwen1) [inline af_trading_bot/core/binance_client.py:9]: Your BinanceClient

**Review by DanielOwen1 — CHANGES_REQUESTED** [review]

**Comment by AledWatkins** [comment]
Thanks Dan, meant to remove that.
Looking at the code this morning, I really didnt like my idea of LogEntry's having LogEvents, I think it was really clumsy and un-needed, so I've taken it out and just left in the LogEntry model.

File: af_trading_bot/core/binance_client.py (hunk 1 above)
Comment (AledWatkins) [reply af_trading_bot/core/binance_client.py:9]: See comment below

File: af_trading_bot/core/models.py [hunk 2]
```diff
@@ -1,3 +1,13 @@
 from django.db import models
 
-# Create your models here.
+
+class LogEntry(models.Model):
+    created_at = models.DateTimeField(auto_now_add=True)
+    message = models.CharField(max_length=2000)
```
Comment (DanielOwen1) [inline af_trading_bot/core/models.py:6]: Could use a TextField here

**Review by DanielOwen1 — APPROVED** [review]

**Pull request merged**